desktop/settings/windows.md

```diff
@@ -269,4 +290,4 @@ Turns on the experimental containerd image store. This brings new features like
 
 ### Experimental features
 
-{% include desktop-experimental.md %}
\ No newline at end of file
+{% include desktop-experimental.md %}
```

Comment:
Trying to find what changed here, or is this just adding a newline at the end? 🤔 😂

Reply:
hmmm :D no idea...